maintainers/maintainer-list.nix

```diff
@@ -13971,4 +13971,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  BarinovMaxim = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
It's already in separated commit

Reply:
but not sorted alphabetically.

Reply:
It isn't important now. Because it's merged https://github.com/NixOS/nixpkgs/pull/164121